eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -48,6 +48,9 @@ parameters:
           - eng/common/TestResources/sub-config/AzurePublicMsft.json
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePaths: 
+          - eng/common/TestResources/sub-config/AzurePublicMsft.json
```

Comment:
```suggestion
        ServiceConnection: azure-sdk-tests-preview
```

Reply:
Thank you for creating this service connection!